influxql/functions.go

```diff
@@ -355,11 +347,19 @@ type FloatHoltWintersReducer struct {
 }
 
 const (
-	defaultAlpha   = 0.5
-	defaultBeta    = 0.5
-	defaultGamma   = 0.5
-	defaultPhi     = 0.5
+	// Arbitrary weight for initializing some intial guesses.
+	// This should be in the  range [0,1]
+	weight = 0.5
```

Comment:
There are enough constants here that I think that would be a good idea. The number of constants is starting to pollute the `influxql` namespace.